tests/src/JIT/Directed/arglist/vararg.cs

```diff
@@ -0,0 +1,1280 @@
+        {
+            long expectedSum = test_passing_longs(expectedValues.Length, __arglist());
+
+            long sum = 0;
+            for (int i = 0; i < expectedValues.Length; ++i)
+            {
+                sum += expectedValues[i];
+            }
+
+            return sum == expectedSum;
+        }
+
+        /// <summary>
+        /// Given an input set create an arglist to pass to a vararg callee.
+        /// 
+        /// The callee will simply loop over the arguments, compute the sum
+        /// then return the value.
+        /// 
+        /// Do a quick check on the value returned, and return whether they are
+        /// equal.
+        /// 
+        /// </summary>
+        /// <param name="expectedValues"></param>
+        /// <returns>bool</returns>
+        static bool TestPassingEmptyFloats(float[] expectedValues)
+        {
+            float expectedSum = test_passing_floats(expectedValues.Length, __arglist());
+
+            float sum = 0;
+            for (int i = 0; i < expectedValues.Length; ++i)
+            {
+                sum += expectedValues[i];
+            }
+
+            return sum == expectedSum;
+        }
+
+        /// <summary>
+        /// Given an input set create an arglist to pass to a vararg callee.
+        /// 
+        /// The callee will simply loop over the arguments, compute the sum
+        /// then return the value.
+        /// 
+        /// Do a quick check on the value returned, and return whether they 
+        /// are equal.
+        /// 
+        /// </summary>
+        /// <param name="expectedValues"></param>
+        /// <returns>bool</returns>
+        static bool TestPassingEmptyDouble(double[] expectedValues)
+        {
+            double expectedSum = test_passing_doubles(expectedValues.Length, __arglist());
+
+            double sum = 0;
+            for (int i = 0; i < expectedValues.Length; ++i)
+            {
+                sum += expectedValues[i];
+            }
+
+            return sum == expectedSum;
+        }
+
+        /// <summary>
+        /// Given an input set create an arglist to pass to a vararg callee.
+        /// 
+        /// The callee will simply loop over the arguments, compute the sum
+        /// then return the value.
+        /// 
+        /// Do a quick check on the value returned, and return whether they are
+        /// equal.
+        /// 
+        /// </summary>
+        /// <param name="expectedValues"></param>
+        /// <returns>bool</returns>
+        static bool TestPassingIntsAndLongs(int[] expectedIntValues, long[] expectedLongValues)
+        {
+            Debug.Assert(expectedIntValues.Length == 2);
+            Debug.Assert(expectedLongValues.Length == 2);
+            long expectedSum = test_passing_int_and_longs(expectedIntValues.Length, expectedLongValues.Length, __arglist(expectedIntValues[0], expectedIntValues[1], expectedLongValues[0], expectedLongValues[1]));
```

Comment:
I can't actually see the values supplied to __arglist(...)
You might want to break up these excessively long lines